app/navigation/LocationOfGoodsNavigator.scala

```diff
@@ -76,7 +76,6 @@ class LocationOfGoodsNavigator @Inject() () extends Navigator {
       case UnlocodeIdentifier            => Some(controllers.locationOfGoods.routes.UnLocodeController.onPageLoad(departureId, mode))
       case AddressIdentifier             => Some(controllers.locationOfGoods.routes.CountryController.onPageLoad(departureId, mode))
       case PostalCodeIdentifier          => Some(controllers.locationOfGoods.routes.PostalCodeController.onPageLoad(departureId, mode))
-      case _                             => None
```

Comment:
Does this not trigger a 'match not exhaustive' exception?

Reply:
**[warn] /Users/josephbowen/repositories/ctc-presentation-notification-frontend/app/navigation/LocationOfGoodsNavigator.scala:71:114: match may not be exhaustive.
[warn] It would fail on the following input: (x: String forSome x not in (AddressIdentifier, AuthorisationNumberIdentifier, CoordinatesIdentifier, CustomsOfficeIdentifier, EoriNumberIdentifier, PostalCodeIdentifier, UnlocodeIdentifier))
[warn]     (userAnswers.get(IdentificationPage) orElse userAnswers.get(InferredIdentificationPage)).map(_.code).flatMap {
[warn]                                                                                                                  ^
[warn] one warning found
[info] compiling 369 Scal**

Reply:
let me fix it